src/helper/billing.js

```diff
@@ -1,5 +1,5 @@
 const stripePackage = require('stripe')(process.env.STRIPE_SECRET_KEY);
-const billing = require('../libs/billing-lib')
+const billing = require('./billing-lib')
```

Comment:
Shouldn't this billing.js file be in handler/ folder?
Then you could rename src/helper/billing-lib.js to src/helper/billing.js.